python/tests/test_tables.py

```diff
@@ -4621,7 +4621,9 @@ def get_msprime_example(self, sample_size, T, seed):
             random_seed=seed,
         )
         ts = tsutil.add_random_metadata(ts, seed)
-        ts = tsutil.insert_random_ploidy_individuals(ts, max_ploidy=1)
+        ts = tsutil.insert_random_ploidy_individuals(
+            ts, max_ploidy=1, samples_only=True
```

Comment:
this is in `TestUnion` - without `samples_only=True` it caused an error, which I worried about, but looked at the situation in detail, and everything is fine (the problem comes from wanting to split up a tree sequence into disjoint bits to then union back together, which is easier if all the individuals are from the same time, and we're not losing generality in the testing here, really)